wolfssl/ssl.h

```diff
@@ -674,6 +674,21 @@ enum AlertLevel {
     alert_fatal   = 2
 };
 
+/* RETURN_CODE macro
+ * Some OpenSSL APIs specify "0" as the return value when an error occurs.
+ * However, some corresponding wolfSSL APIs(eg.
+ * wolfSSL_CTX_load_verify_locations) return negative values. Such functions
+ * should use this macro to fill this gap. Users who want them to return
+ * the same return value as OpenSSL can define WOLFSSL_ERR_CODE_OPENSSL.
+ * Note that this macro replaces only negative return values with the
+ * specified value.
+ */
+#if defined(WOLFSSL_ERROR_CODE_OPENSSL)
+    #define RETURN_CODE(w,o)  ((w < 0)?o:w)
```

Comment:
RETURN_CODE is too generic.  Likely to cause conflicts at some point.  Please add a WS prefix or anything.

Don't use o and 0 in the same macro and use spaces between ? item1 : item2 for readability.

Reply:
Changed macro name to WS_RETURN_CODE and also changed parameter names.